site/_posts/2018/2018-11-07-whats-new-in-php-ml-0-7-0.md

```diff
@@ -1,11 +1,11 @@
 ---
-layout: post
+id: 2
```

Comment:
Every post needs own `id` to better work with inter-connections, like `relatedItems` filter etc.